skycultures/modern_iau/index.json

```diff
@@ -7,442 +7,442 @@
+      "common_name": {"english": "Keel", "native": "Carina", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Cas",
       "lines": [[746, 3179, 4427, 6686, 8886]],
-      "common_name": {"english": "Cassiopeia", "native": "Cassiopeia"}
+      "common_name": {"english": "Seated Queen", "native": "Cassiopeia", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Cen",
       "lines": [[61932, 66657, 68702, 71683, 68702, 66657, 68002, 61932, 68002, 68282, 68245, 71352, 68245, 68862, 70090, 68933, 67464, 68002], [55425, 59196, 60823, 61932, 60823, 59449, 56243], [67464, 65936, 65109, 61789], [71352, 73334]],
-      "common_name": {"english": "Centaurus", "native": "Centaurus"}
+      "common_name": {"english": "Centaur", "native": "Centaurus", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Cep",
       "lines": [[102422, 105199, 106032, 112724, 106032, 116727, 112724, 110991, 109492, 109857, 107259, 105199], [101093, 102422]],
-      "common_name": {"english": "Cepheus", "native": "Cepheus"}
+      "common_name": {"english": "King", "native": "Cepheus", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Cet",
       "lines": [[12706, 14135, 13954, 12828, 11484, 12706, 12387, 10826, 8645, 8102, 3419, 1562, 5364, 6537, 8645]],
-      "common_name": {"english": "Cetus", "native": "Cetus"}
+      "common_name": {"english": "Sea Monster", "native": "Cetus", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Cha",
       "lines": [[40702, 51839, 52633, 60000, 58484, 51839]],
-      "common_name": {"english": "Chamaeleon", "native": "Chamaeleon"}
+      "common_name": {"english": "Chameleon", "native": "Chamaeleon", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Cir",
       "lines": [[74824, 71908, 75323]],
-      "common_name": {"english": "Circinus", "native": "Circinus"}
+      "common_name": {"english": "Drawing Compass", "native": "Circinus", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Cnc",
       "lines": [[44066, 42911, 40526, 42911, 42806, 43103]],
-      "common_name": {"english": "Cancer", "native": "Cancer"}
+      "common_name": {"english": "Crab", "native": "Cancer", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Col",
       "lines": [[26634, 27628], [25859, 26634], [28328, 27628, 28199, 30277]],
-      "common_name": {"english": "Columba", "native": "Columba"}
+      "common_name": {"english": "Dove", "native": "Columba", "context": "IAU constellation name"}
     },
     {
       "id": "CON modern_iau Com",
       "lines": [[64241, 64241, 64394, 60742]],
-      "common_name": {"english": "Coma Berenices", "native": "Coma Berenices"}
+      "common_name": {"english": "Bernice's Hair", "native": "Coma Berenices", "context": "IAU constellation name"}
```

Comment:
Is it really "Bernice", not "Berenice" in English?

Reply:
Yeah, see https://www.iau.org/public/themes/constellations/